packages/cli/bin/ak-dev.js

```diff
@@ -49,7 +49,7 @@ program.addHelpText('after', () => {
   $ ak dev -p 8888
 
   ${chalk.gray('# run ak dev server without HMR')}
-  $ ak dev --no-hotReplace
+  $ ak dev --no-hot-replace
```

Comment:
```suggestion
  $ ak dev --no-hotReplace
```
look this line https://github.com/myNameIsDu/aktiv/pull/76/files#diff-ff0097cf4bb6fc14400937827586e93b92767108ee38e81b6ace045faf33b6ffL180

Reply:
yeah

Reply:
so i want to use "hot-replease"

Reply:
how do you think?

Reply:
<img width="635" alt="image" src="https://user-images.githubusercontent.com/39019913/158061304-3e54cc03-6431-441b-aff3-88b7da13441b.png">